docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3'
```

Comment:
Remove this file. All compose files are in edgex-compose. A new option will be assed to the compose builder for this device service once it have been accepted in to `edgexfoundry`